src/lib.rs

```diff
@@ -3,7 +3,6 @@
 #![warn(unreachable_pub)]
 #![recursion_limit = "256"]
 #![allow(clippy::match_like_matches_macro)]
-#![allow(unreachable_pub)]
```

Comment:
IIRC this was added upstream at one point when it was necessary due to some changes being applied in r-l and could've been removed after a subtree sync, so doing it now